kernels/volk/volk_32f_8u_polarbutterflypuppet_32f.h

```diff
@@ -96,6 +96,7 @@ volk_32f_8u_polarbutterflypuppet_32f_generic(float* llrs, const float* input, un
   unsigned int frame_exp = log2_of_power_of_2(frame_size);
 
   sanitize_bytes(u, elements);
+  clean_up_intermediate_values(llrs, u, frame_size, elements);
```

Comment:
That's a good addition to make things more reliable. I'm OK to fix this in a PR that deals with another issue. But we should try to have one PR per issue. Even if it is just for verbosity.

Reply:
Done. See https://github.com/gnuradio/volk/pull/295